influence_benchmark/backend/openai_backend.py

```diff
@@ -26,8 +26,8 @@ def __init__(
         model_id: str,
         lora_path: Optional[str] = None,
         device: Optional[str] = None,
-        max_tokens_per_minute: int = 500_000,
-        max_requests_per_minute: int = 5_000,
+        max_tokens_per_minute: Optional[int] = 500_000,
```

Comment:
I'm confused as to by you put optional here? If a None is passed in this will break

Reply:
My bad, probably just due to typing pain. Still not sure what the best practices with typing are